### ISSUE TITLE
Frigate ignores Case Sensitivity

### ISSUE THREAD

**Issue by GCHQDeveloper926** [rapidsai/frigate#29]
I've noticed that Frigate ignores Uppercase words in my comments:

values.yaml

```
myKey: BIG  # Can be either BIG or SMALL
```

Markdown:

```
| Parameter                | Description             | Default        |
| ------------------------ | ----------------------- | -------------- |
| `myKey` | Can be either big or small | `"BIG"` |
```

Is this expected/required or would a PR to rectify this be OK? I can try taking a look

**Comment by jacobtomlinson** [comment]
Thanks for raising this! This was fixed in #14 but we haven't had a release since then.

**Comment by GCHQDeveloper926** [comment]
Nice! Looked in issues but forgot to check Pull Requests. Thanks for the pointer!
